public/locale/en/translations.json

```diff
@@ -1,5 +1,8 @@
 {
-  "common": {
-    "welcomeMessage": "Welcome World!"
+  "auth": {
+    "login": "Login",
+    "emailAddress": "EMAIL ADDRESS",
+    "password": "PASSWORD",
```

Comment:
Why are these strings all caps ?

Reply:
So these strings are the placeholders of login screen input. We can make it camel case, will update these.